doc/contribute/external.rst

```diff
@@ -164,3 +170,49 @@ The flowchart below shows an overview of the process:
    :align: center
 
    Submission process
+
+.. _external-tooling:
+
+Contributing External Tooling
+*****************************
+
+This section deals exclusively with the inclusion of external tooling in the
+Zephyr project, where tooling is defined as software that assists the
+compilation, testing or simulation processes but in no case ends up being part
+of the code compiled and linked into the final image.
+
+Tooling components must be released under a license approved by the
+`Open Source Initiative (OSI)`_.
+
+Just like with regular external components, tooling that is imported from
+another project can be integrated either in the main tree or as a :ref:`west
+project <west-workspace>`. Note that in this case the corresponding west project
+will not be a :ref:`module <modules>`, because tooling does not make use of the
+Zephyr build system and does not need to be processed by it. Please see
+:ref:`modules-vs-projects` for additional information on the differences.
+
+If the tool is integrated in the main tree it should be placed under the
+:file:`scripts/` folder.
+If the tool is integrated as a west project, then the project repository can be
+hosted outside the zephyrproject-rtos GitHub organization, provided that the
+project is made optional via the `group-filter:` field in the main `west.yml`
+manifest. More info on optional projects can be found in :ref:`this section
+<west-manifest-groups>`.
+
+The TSC must approve every Pull Request that introduces a new external tooling
+component. This will be done on a case-by-case, individual analysis of the
+proposed addition by the TSC representatives.
```

Comment:
I still think we should describe more about what kinda of tooling would be found to be acceptable and what might not.

Reply:
I think its about giving an example.  That something like a script utilizing pyelf is ok since its OSI approved, etc...